scripting/viewmodel_override.sp

```diff
@@ -20,13 +20,15 @@
 #pragma newdecls required
 
 #include <tf_custom_attributes>
+#include <cwx>
```

Comment:
Mind migrating the implementation to only use the custom attribute system?  While the repo is intended to have attributes for CWX to use, I do still want them to be usable standalone.

`TF2CustAttr_GetAttributeKeyValues` is also available and subsections are allowed, so you should be able to implement it in a similar way; just sub the item UID check for checking that the subsections exist.

Reply:
did i do it right?

Reply:
Should be fine, but I'll test it shortly and fixup some other minor nits.